src/js/resize-manager.js

```diff
@@ -59,21 +59,22 @@ class ResizeManager extends Component {
     this.resizeObserver_ = null;
     this.debouncedHandler_ = debounce(() => {
       this.resizeHandler();
-    }, 100, false, player);
+    }, 100, false, this);
```

Comment:
seemed weird to put this into the players context, but then call a function in our context.  This probably isn't related to the bug though.